streams/shared/src/main/scala/zio/stream/Stream.scala

```diff
@@ -225,6 +226,30 @@ object Stream extends Serializable {
   final def fromIteratorManaged[E, A](iterator: Managed[E, Iterator[A]]): Stream[E, A] =
     ZStream.fromIteratorManaged(iterator)
 
+  /**
+   * See [[ZStream.fromJavaIterator]]
+   */
+  final def fromJavaIterator[E, A](iterator: IO[E, ju.Iterator[A]]): Stream[E, A] =
```

Comment:
Shouldn't these go under the `jvm` branch?

Reply:
Hmm it compiles on JS but that might be misleading. I’ll check if Iterator and Stream are implemented there.

Reply:
So there's a ju.Iterator implementation but not Stream. Weird that it compiles. Anyway I moved the Stream things to platform-specific.

Reply:
What about this? Not very familiar with converters though. Maybe you could take a look.
https://www.scala-lang.org/api/2.13.1/scala/jdk/StreamConverters$.html

Reply:
Ah, maybe it's only available for 2.13. :/